RELEASING.md

```diff
@@ -0,0 +1,244 @@
+# MusicBrainz Server release process
+
+> Preamble:
+> This document covers steps for releasing new versions of MusicBrainz Server
+> which is performed by maintainers only.
+> It includes discussion about private servers, repositories and tools which
+> other contributors don’t have access to.
+> It is made public for transparency and to allow for improvement suggestions.
+
+## Table of contents
+
+<!-- toc -->
+
+- [Prerequisites](#prerequisites)
+- [Release production](#release-production)
+  * [Update translated messages](#update-translated-messages)
+  * [Merge Git branches](#merge-git-branches)
+  * [Build Docker images](#build-docker-images)
+  * [Announce the deployment](#announce-the-deployment)
+  * [Deploy to production](#deploy-to-production)
+  * [Update websites’ banner](#update-websites-banner)
+  * [Update Jira](#update-jira)
+  * [Blog](#blog)
+  * [Add Git tag](#add-git-tag)
+  * [Release musicbrainz-docker](#release-musicbrainz-docker)
+- [Release beta](#release-beta)
+- [Update test](#update-test)
+
+<!-- tocstop -->
+
+## Prerequisites
+
+1. Perl module Locale::PO
+
+   For updating translations, also part of `production` server features in `cpanfile`.
+
+2. Transifex client
+
+   See the section “[Translations](INSTALL.md#translations)” of the installation documentation.
+
+See the private system administration wiki for additional prerequisites.
+
+## Release production
+
+### Update translated messages
+
+Assuming the source messages were updated when releasing beta (if not, see that under the beta
+process below first!), you need to start by updating the translated messages:
+
+1. Run `./po/update_translations.sh --commit` to download the latest .po files
+   from Transifex and commit them to the `beta` branch.
+
+### Merge Git branches
+
+1. Merge `beta` to `production` (`git merge --log=876423 --no-ff beta`) and push.
+   Wait until [CircleCI](https://circleci.com/gh/metabrainz/musicbrainz-server) and
+   [Jenkins (Selenium)](https://ci.metabrainz.org/job/musicbrainz-server/) are happy
+   with this merge.
+
+2. Merge `production` to `master` and push.
+
+### Build Docker images
+
+Then, you must build new MusicBrainz Server Docker images from Jenkins, go to
+
+1. Go to https://ci.metabrainz.org/job/musicbrainz-docker-images/
```

Comment:
go to: 1. Go to  :) Maybe just do:
```suggestion
Then, you must build new MusicBrainz Server Docker images from Jenkins:

1. Go to https://ci.metabrainz.org/job/musicbrainz-docker-images/
```

Reply:
Committed as https://github.com/metabrainz/musicbrainz-server/pull/2340/commits/8b1f15982cac04676edbd628fd00315ea60fab12